qSevenIndex.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html >
+    <head>
+        <title>Question Seven</title>
+        <script src="./qSevenIndex.js"></script>
+    </head>
+    <body>
+        <input id="number" type="number"/><br>
+        <button id="submitButton" type="submit"> Submit</button>
+        
+        <ol>
+             Numbers
+        </ol>
```

Comment:
The <ol> tag defines an ordered list. You need items inside, use the <li> tag to define list items.